src/reynir_correct/errtokenizer.py

```diff
@@ -1512,25 +1537,30 @@ class MultiwordErrorStream(MatchingStream):
     and inserting replacement phrases when matches are found"""
 
     def __init__(
-        self, db: GreynirBin, token_ctor: TokenCtor, ignore_rules: FrozenSet[str]
+        self,
+        db: GreynirBin,
+        token_ctor: TokenCtor,
+        ignore_rules: FrozenSet[str],
+        settings,
     ) -> None:
-        super().__init__(MultiwordErrors.DICT)
+        super().__init__(settings.multiword_errors.DICT)
         self._token_ctor = token_ctor
         self._db = db
         self._ignore_rules = ignore_rules
+        self.settings = settings
```

Comment:
Could be more efficient to assign a local variable and use it, instead of multiple lookups:

`self.multiword_errors = settings.multiword_errors`

Reply:
Fixed